scripts/preprocess/topo_county_boundaries.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash  
+
+# create a temp directory
+TMP=$(mktemp -d)
+
+# unzip the geojson
+unzip $1 -d $TMP
+
+# pick out the geojson files (exclude counties.js and states.js)
+GJ=$(dir $TMP/*.geojson)
+
+# list state fips for now
+while read fip
```

Comment:
neat! good job getting this bash loop figured out.

Reply:
the real mvp: @wdwatkins

Reply:
:100: